gobject-subclass/src/properties.rs

```diff
@@ -16,6 +16,8 @@ pub enum PropertyMutability {
     Readable,
     Writable,
     ReadWrite,
+    Construct,
+    ConstructOnly
```

Comment:
This should be in a separate commit, and has to be solved a bit differently. `Construct` properties can be read and/or write, `ConstructOnly` properties can be readable (after construction) or they can be neither. In C these are all flags that can be combined in arbitrary ways.

The rules are here https://gitlab.gnome.org/GNOME/glib/blob/a65ee63c711111d61deb336da9ed9a13ce0126e8/gobject/gobject.c#L539-554, apart from that it can be combined in any possible way

Reply:
I figure we need the bitflags crate here?

Reply:
Or we express the possible combinations with one or two enums. Not all combinations are valid, so bitflags are not so nice either.

Reply:
Oh I see the problem... hm!

Reply:
Sure, it's only for the test anyway